### PR TITLE
fix(DB/GO): Sha'naar Relic (9430) - 181637 - incorrect spawn xyz

### PR DESCRIPTION
**Pull request by wowquester**
## Changes Proposed:
-  corrected Z coordinate for 181637 (for guid 22070) - now spaws correctly on the ground level instead of under it

## Issues Addressed:

- Closes https://github.com/chromiecraft/chromiecraft/issues/5899

## SOURCE:

proof1_offi
proof2_offi

## Tests Performed:

Before 
https://user-images.githubusercontent.com/138876396/252259369-cf9cbdf0-a6c4-4baf-a86e-8a66b853c931.jpg
After
WoWScrnShot_071023_090558

## How to Test the Changes:

1. .q a 9430
2. .go xyz
3. .go xyz -521.574341 4874.057617 34.331928

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

File: data/sql/updates/pending_db_world/fixq9430.sql [hunk 1]
```diff
@@ -0,0 +1,3 @@
+DELETE FROM `gameobject` WHERE (`id` = 181637) AND (`guid` IN (22070));
+INSERT INTO `gameobject` (`guid`, `id`, `map`, `zoneId`, `areaId`, `spawnMask`, `phaseMask`, `position_x`, `position_y`, `position_z`, `orientation`, `rotation0`, `rotation1`, `rotation2`, `rotation3`, `spawntimesecs`, `animprogress`, `state`, `ScriptName`, `VerifiedBuild`) VALUES
+(22070, 181637, 530, 3483, 3551, 1, 1, -521.589, 4872.55, 34.194775, -2.09439, 0, 0, 0.866025, -0.5, 181, 100, 1, '', 0);
```
Comment (Helias) [inline data/sql/updates/pending_db_world/fixq9430.sql:3]: same here, the name of the file is wrong, please follow this documentation for it https://www.azerothcore.org/wiki/sql-versioning

Comment (wowquester) [reply data/sql/updates/pending_db_world/fixq9430.sql:3]: Hey Helias, sorry about that... in this case, shouldn't it be linked to the wiki? I am new to this so I just followed the wiki. Maybe I got it wrong.
obrazek
But at the same time I don't know how to properaly fix those issues - i don't understand the versioning - i understand it is a convention, a very needed one, but not sure how to implement it... if you find a minute, do you think you could help? -or explain? The wiki is too technical and it seems like i'm missing a technical background for it.

Comment (wowquester) [reply data/sql/updates/pending_db_world/fixq9430.sql:3]: date +%s%N - how to do it, since I'm on windows...

Comment (Helias) [reply data/sql/updates/pending_db_world/fixq9430.sql:3]: do not hesitate to comment or ask via discord tagging me if you encounter any issue ;-)

**Review by wowquester — COMMENTED** [review]
Review, please review as well..

**Comment by wowquester** [comment]
Ok, now it's failing. I have no idea what is happening....

**Review by Helias — APPROVED** [review]

File: data/sql/updates/pending_db_world/rev_1689005068014101100.sql [hunk 2]
```diff
@@ -0,0 +1,4 @@
+INSERT INTO `version_db_auth` (`sql_rev`) VALUES ('1689005171898848000');
```
Comment (Helias) [inline data/sql/updates/pending_db_world/rev_1689005068014101100.sql:1]: this should not be here If I remember correctly, indeed the error in the github pipeline is the following 
```
ERROR 1146 (42S02) at line 2 in file: '/home/runner/work/azerothcore-wotlk/azerothcore-wotlk/data/sql/updates/db_world/2023_07_10_03.sql': Table 'acore_world.version_db_auth' doesn't exist
```
```suggestion
```

**Review by Helias — APPROVED** [review]

**Comment by heyitsbench** [comment]
Few things I should say:
1) If all that's being changed for the gobject is the position, I'd recommend just using an update query for those specific offending values. Is there a particular reason for DELETE/INSERT over that in this case?
2) Missing newline at the end of the SQL file. You can tell (in Github web view at least) by the ⛔ symbol at the end of the [file](https://github.com/azerothcore/azerothcore-wotlk/pull/16725/files). No idea why the workflow is fine with it, but at least in my experience newlines are required at the end of SQL query files.
3) Gobject position data is a sniffed value. I've attached the appropriate label.

File: data/sql/updates/pending_db_world/fixq9430.sql (hunk 1 above)
Comment (Kitzunu) [reply data/sql/updates/pending_db_world/fixq9430.sql]: file name doesnt matter. CI will rename it when importing. See https://github.com/azerothcore/azerothcore-wotlk/commit/27943b6175ca6c2d95d5db9d7bfd8e550f8a2d72

Comment (Helias) [reply data/sql/updates/pending_db_world/fixq9430.sql]: oh ok, sorry then ;-)

**Comment by Kitzunu** [comment]
Should be sniffed.

**Comment by Gultask** [comment]
If TC or Mangos is better you can also use the same values from them

**Comment by Kitzunu** [comment]
TC has same values as stock AC

**Comment by Kitzunu** [comment]
https://github.com/azerothcore/azerothcore-wotlk/pull/18188

**Pull request closed without merging**